staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate.go

```diff
@@ -344,6 +351,28 @@ func (o AnnotateOptions) RunAnnotate() error {
 			}
 		}
 
+		if o.list {
+			accessor, err := meta.Accessor(outputObj)
+			if err != nil {
+				return err
+			}
+
+			indent := ""
+			if !singleItemImpliedResource {
+				indent = " "
+				gvks, _, err := unstructuredscheme.NewUnstructuredObjectTyper().ObjectKinds(info.Object)
+				if err != nil {
+					return err
+				}
+				fmt.Fprintf(o.ErrOut, "Listing annotations for %s.%s/%s:\n", gvks[0].Kind, gvks[0].Group, info.Name)
```

Comment:
Not a huge fan of using stderr for informational messages, but I know this is the [same way `label --list` does it](https://github.com/kubernetes/kubernetes/blob/908847c01e9640ffce2ffda5acf88d92c48a5148/staging/src/k8s.io/kubectl/pkg/cmd/label/label.go#L365).

I'll leave this up to @soltysh to comment on, but since it is done the same way in `label --list`, I'm fine with either leaving it this way, or fixing it in both places.

Reply:
Thanks, Done in https://github.com/kubernetes/kubernetes/pull/92691 :)